ui/components/link.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { Link as RouterLink, routes } from '../routes';
+
+const existingRoutes = routes.map(r => r.name);
+
+export default function Link({ children, className, href, params, route }) {
+  if (route) {
+    return (
+      <RouterLink params={params} route={route}>
+        <a className={className}>{children}</a>
+      </RouterLink>
+    );
+  } else if (href) {
+    const attrs = href.search(/^https|http|^\/\//) === 0 ? {
+      target: '_blank',
+      rel: 'noopener noreferrer',
+      'aria-label': 'Link opens in a new window',
+    } : {};
+    return (<a className={className} href={href} {...attrs} >{children}</a>);
+  }
+}
+
+/* eslint-disable consistent-return, react/forbid-prop-types */
+Link.propTypes = {
+  children: PropTypes.node.isRequired,
+  className: PropTypes.string,
+  href: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
```

Comment:
When would this be a boolean?

Reply:
It won't anymore, but I had the default prop as `false` to make the conditional easy, but that conditional will still work with the new default `null`. I'll update this.